javascript/ql/src/Security/CWE-830/CUSTOMIZING.md

```diff
@@ -0,0 +1,43 @@
+# Functionaility from untrusted source/domain - customizations
+
+You can extend the behavior of the `js/functionality-from-untrusted-source` and `js/functionality-from-untrusted-domain` queries using [CodeQL data extensions](https://codeql.github.com/docs/codeql-language-guides/customizing-library-models-for-javascript/).
+
+This allows you to require Subresource Integrity (SRI) checks on specific content delivery network (CDN) hostnames, and add additional domains to warn on, respectively.
```

Comment:
I'm not sure that "respectively" adds anything in this case.

As a non-expert, it seems to me as if step one is adding more domains and step 2 is requiring SRI check, so I'd tend to suggest swapping the sentence around.

```suggestion
This allows you to add additional domains to warn users about and to require Subresource Integrity (SRI) checks on specific content delivery network (CDN) hostnames.
```